runtime/cere-dev/src/hyperbridge_ismp.rs

```diff
@@ -0,0 +1,227 @@
+	pub const NativeAssetId: u32 = 0;
+	// Set the correct decimals for the native currency
+	pub const Decimals: u8 = 10;
+}
+
+/// Should provide an account that is funded and can be used to pay for asset creation
+pub struct AssetAdmin;
+impl Get<AccountId> for AssetAdmin {
+	fn get() -> AccountId {
+		Treasury::account_id()
+	}
+}
+
+pub struct MockAssets;
+
+impl fungibles::metadata::Inspect<AccountId> for MockAssets {
+	fn name(_asset: Self::AssetId) -> Vec<u8> {
+		vec![]
+	}
+
+	fn symbol(_asset: Self::AssetId) -> Vec<u8> {
+		vec![]
+	}
+
+	fn decimals(_asset: Self::AssetId) -> u8 {
+		0
+	}
+}
+impl Unbalanced<AccountId> for MockAssets {
+	fn handle_dust(_dust: Dust<AccountId, Self>) {}
+
+	fn write_balance(
+		_asset: Self::AssetId,
+		_who: &AccountId,
+		_amount: Self::Balance,
+	) -> Result<Option<Self::Balance>, DispatchError> {
+		Ok(None)
+	}
+
+	fn set_total_issuance(_asset: Self::AssetId, _amount: Self::Balance) {}
+}
+
+impl fungibles::Mutate<AccountId> for MockAssets {}
+
+impl Inspect<AccountId> for MockAssets {
+	type AssetId = Balance;
+	type Balance = Balance;
+
+	fn total_issuance(_asset: Self::AssetId) -> Self::Balance {
+		0
+	}
+
+	fn minimum_balance(_asset: Self::AssetId) -> Self::Balance {
+		0
+	}
+
+	fn total_balance(_asset: Self::AssetId, _who: &AccountId) -> Self::Balance {
+		0
+	}
+
+	fn balance(_asset: Self::AssetId, _who: &AccountId) -> Self::Balance {
+		0
+	}
+
+	fn reducible_balance(
+		_asset: Self::AssetId,
+		_who: &AccountId,
+		_preservation: Preservation,
+		_force: Fortitude,
+	) -> Self::Balance {
+		0
+	}
+
+	fn can_deposit(
+		_asset: Self::AssetId,
+		_who: &AccountId,
+		_amount: Self::Balance,
+		_provenance: Provenance,
+	) -> DepositConsequence {
+		DepositConsequence::UnknownAsset
+	}
+
+	fn can_withdraw(
+		_asset: Self::AssetId,
+		_who: &AccountId,
+		_amount: Self::Balance,
+	) -> WithdrawConsequence<Self::Balance> {
+		WithdrawConsequence::UnknownAsset
+	}
+
+	fn asset_exists(_asset: Self::AssetId) -> bool {
+		false
+	}
+}
+
+impl fungibles::Create<AccountId> for MockAssets {
+	fn create(
+		_id: Self::AssetId,
+		_admin: AccountId,
+		_is_sufficient: bool,
+		_min_balance: Self::Balance,
+	) -> DispatchResult {
+		Ok(())
+	}
+}
+
+impl fungibles::metadata::Mutate<AccountId> for MockAssets {
+	fn set(
+		_asset: Self::AssetId,
+		_from: &AccountId,
+		_name: Vec<u8>,
+		_symbol: Vec<u8>,
+		_decimals: u8,
+	) -> frame_support::dispatch::DispatchResult {
+		Ok(())
+	}
+}
+
+impl fungibles::roles::Inspect<AccountId> for MockAssets {
+	fn owner(_asset: Self::AssetId) -> Option<AccountId> {
+		None
+	}
+
+	fn issuer(_asset: Self::AssetId) -> Option<AccountId> {
+		None
+	}
+
+	fn admin(_asset: Self::AssetId) -> Option<AccountId> {
+		None
+	}
+
+	fn freezer(_asset: Self::AssetId) -> Option<AccountId> {
+		None
+	}
+}
+impl pallet_token_gateway::Config for Runtime {
+	// configure the runtime event
+	type RuntimeEvent = RuntimeEvent;
+	// Configured as Pallet Ismp
+	type Dispatcher = Ismp;
```

Comment:
use `pallet-hyperbridge` as your dispatcher

```suggestion
	type Dispatcher = pallet_hyperbridge::Pallet<Runtime>;
```